vignettes/extract-fotmob-data.Rmd

```diff
@@ -282,3 +282,12 @@ salah_shotmap <- salah %>%
   tidyr::unnest(shotmap)
 dplyr::glimpse(salah_shotmap)
 ```
+
+Note that `stats_` columns are returned as characters (**changed in `0.5.9`**).
+
+```{r salah_stats}
+salah_stats <- salah %>% 
+  dplyr::select(player_id = id, tidyselect::vars_select_helpers$starts_with("stats_")) %>% 
+  janitor::remove_empty(which = "cols")
+dplyr::glimpse(salah_stats)
+```
```

Comment:
this is the only actual change in this PR. everything else is the result of running `pkgdown::build_site()`